sec.club/src/views/SLP/Projects.js

```diff
@@ -0,0 +1,30 @@
+import openTogetherTubeLogo from "../../assets/project-wall/open-together-tube-logo.svg"
+import secLogo from "../../assets/logo.svg"
+
+const PROJECTS = [
+  {
+    path: "quick-quack",
+    img: secLogo,
+    name: "Quick Quack",
+    semester: "Spring",
+    year: "2019",
+    markdown: "slp/quick-quack.md"
+  },
+  {
+    path: "open-together-tube",
+    img: openTogetherTubeLogo,
+    name: "Open Together Tube",
+    semester: "Fall",
+    year: "2019",
+    markdown: "slp/open-together-tube.md"
+  },
+  {
+    path: "stevens-web-extension",
+    img: secLogo,
+    name: "Stevens Web Extension",
+    semester: "Spring",
+    year: "2020",
+    markdown: "slp/quick-quack.md"
```

Comment:
Why does the stevens web extension reference the quick-quack markdown file?

Reply:
Didn't bother to change it. We don't have text content for it yet anyway

Reply:
Fair enough